docs/release_notes.md

```diff
@@ -4,11 +4,15 @@
 
 ### Notable changes
 
-- Root certificate rotation is now supported through the new CRD, MeshRootCertificate.
+- OSM certificate provider is now configured using the new CRD, MeshRootCertificate
+  - Custom trust domains (i.e. certificate CommonNames) are now supported
```

Comment:
@steeling Just want to confirm this is accurate

Reply:
I thought we are no longer using the CRD?

Reply:
See https://github.com/openservicemesh/osm/pull/4865#discussion_r914183380